src/main/java/com/iridium/iridiumskyblock/gui/ShopCategoryGUI.java

```diff
@@ -112,35 +114,36 @@ public void onInventoryClick(InventoryClickEvent event) {
     }
 
     private void addShopLore(List<String> lore, ShopItem item) {
+        Shop shopConfig = IridiumSkyblock.getInstance().getShop();
         if (item.isPurchasable()) {
             lore.add(
-                    StringUtils.color(IridiumSkyblock.getInstance().getShop().buyPriceLore
+                    StringUtils.color(shopConfig.buyPriceLore
                             .replace("%amount%", String.valueOf(item.defaultAmount))
                             .replace("%buy_price_vault%", String.valueOf(item.buyCost.vault))
                             .replace("%buy_price_crystals%", String.valueOf(item.buyCost.crystals))
                     )
             );
         } else {
-            lore.add(StringUtils.color(IridiumSkyblock.getInstance().getShop().notPurchasableLore));
+            lore.add(StringUtils.color(shopConfig.notPurchasableLore));
         }
 
         if (item.isSellable()) {
             lore.add(
-                    StringUtils.color(IridiumSkyblock.getInstance().getShop().sellRewardLore
+                    StringUtils.color(shopConfig.sellRewardLore
                             .replace("%amount%", String.valueOf(item.defaultAmount))
                             .replace("%sell_reward_vault%", String.valueOf(item.sellReward.vault))
                             .replace("%sell_reward_crystals%", String.valueOf(item.sellReward.crystals))
                     )
             );
         } else {
-            lore.add(StringUtils.color(IridiumSkyblock.getInstance().getShop().notSellableLore));
+            lore.add(StringUtils.color(shopConfig.notSellableLore));
         }
 
-        IridiumSkyblock.getInstance().getShop().shopItemLore.stream()
+        lore.addAll(shopConfig.shopItemLore.stream()
                 .map(StringUtils::color)
-                .forEach(line -> lore.add(
-                        line.replace("%amount%", String.valueOf(item.defaultAmount))
-                ));
+                .map(line -> line.replace("%buylore%", item.isPurchasable() ? shopConfig.shopItemLoreBuy : ""))
+                .map(line -> line.replace("%selllore%", item.isSellable() ? shopConfig.shopItemLoreSell : ""))
```

Comment:
This will cause empty lines, won't it?

Reply:
ye it just does empty line but at leats fixes :panda:
I didn't want make it add lore cuz may users wants change lore